packages/neon-api/__tests__/transaction/builder.ts

```diff
@@ -0,0 +1,60 @@
+import { TransactionBuilder } from "../../src/transaction";
+import { tx, u } from "@cityofzion/neon-core";
+
+describe("constructor", () => {
+  test("no param", () => {
+    const txBuilder = new TransactionBuilder();
+    expect(txBuilder.build().script).toBe("");
+  });
+
+  test("with param", () => {
+    const txBuilder = new TransactionBuilder({ script: "abcd" });
+    expect(txBuilder.build().script).toBe("abcd");
+  });
+});
+
+describe("setter", () => {
+  test("addAttributes", () => {
+    const transaction = new TransactionBuilder()
+      .addAttributes({
+        usage: 129,
+        data: "abcd"
+      })
+      .build();
+    expect(transaction.attributes[0].data).toBe("abcd");
```

Comment:
please use the equality method for all the tests instead of comparing only a single field. Also test adding multiple items correctly.